src/Sylius/Bundle/ApiBundle/CommandHandler/Checkout/SendOrderConfirmationHandler.php

```diff
@@ -14,19 +14,18 @@
 namespace Sylius\Bundle\ApiBundle\CommandHandler\Checkout;
 
 use Sylius\Bundle\ApiBundle\Command\Checkout\SendOrderConfirmation;
-use Sylius\Bundle\CoreBundle\Mailer\Emails;
+use Sylius\Bundle\CoreBundle\Mailer\OrderEmailManagerInterface;
 use Sylius\Component\Core\Model\OrderInterface;
 use Sylius\Component\Core\Repository\OrderRepositoryInterface;
-use Sylius\Component\Mailer\Sender\SenderInterface;
 use Symfony\Component\Messenger\Handler\MessageHandlerInterface;
 use Webmozart\Assert\Assert;
 
 /** @experimental */
 final class SendOrderConfirmationHandler implements MessageHandlerInterface
 {
     public function __construct(
-        private SenderInterface $emailSender,
         private OrderRepositoryInterface $orderRepository,
+        private OrderEmailManagerInterface $orderEmailManager,
```

Comment:
From what I see, this change isn't mentioned in the upgrade file. Is it on purpose? It also applies to the file below.